examples/tensorflow/language-modeling/run_clm.py

```diff
@@ -260,9 +258,9 @@ def main():
         training_args.output_dir = Path(training_args.output_dir)
         os.makedirs(training_args.output_dir, exist_ok=True)
 
-    if isinstance(training_args.strategy, tf.distribute.TPUStrategy) and not data_args.pad_to_max_length:
-        logger.warning("We are training on TPU - forcing pad_to_max_length")
-        data_args.pad_to_max_length = True
+    if isinstance(training_args.strategy, tf.distribute.TPUStrategy) and not data_args.pad_to_block_size:
```

Comment:
I don't see where this would be used currently. Also, I am not testing on a TPU, so I'm not sure this would work, but surely it didn't before this commit.

Reply:
This and the `pad_to_max_length` argument should all be removed since it's not used anywhere.

Reply:
Agreed! This section was incorrectly copied from other examples, where variable batch lengths cause chaos on TPUs. Obviously not going to happen in CLM, my bad.